manifests/RProject/R/4.0.0.yaml

```diff
@@ -0,0 +1,12 @@
+Id: RProject.R
+Publisher: RProject
+Name: RProject R
+Version: 4.0.0
+License: GNU General Public License, version 2
+LicenseUrl: https://www.r-project.org/Licenses/
+Description: The base binary distribution of R statistics language.
+InstallerType: exe
+Installers:
+  - Arch: x64
```

Comment:
This installer also works for x86 architecture (32bit). Maybe worth adding also ?

Reply:
I'm looking into it. Not sure exactly how this can be done, considering that every manifest I can find either use ```x86``` or ```x64``` with the exception of Rufus that has the attribute ```Neutral```, but I'm not sure exactly what this mean.

Reply:
I though a manifest can have several link, one per architecture... 🤔 
Using `WinGetYamlGenerator` from the Store, I was offered to add several binaries and it generated this 
```yaml
Installers:
  - Arch: x64
    InstallerType: EXE
    Url: https://cloud.r-project.org/bin/windows/base/R-4.0.0-win.exe
    Sha256: CA0C28A4F3E0AE66F64F01024BCCF3F3477C6E1008698AB96CEB3162E24E6C87
  - Arch: x86
    InstallerType: EXE
    Url: https://cloud.r-project.org/bin/windows/base/R-4.0.0-win.exe
    Sha256: CA0C28A4F3E0AE66F64F01024BCCF3F3477C6E1008698AB96CEB3162E24E6C87
```
So I thought it was supported

Reply:
From the specification, seems ok.
https://docs.microsoft.com/fr-fr/windows/package-manager/package/manifest?tabs=minschema%2Ccompschema

Reply:
Tnx. I think this will work.

Reply:
This might not be allowed at the moment. The local validation didn't indicate anything wrong, but the Azure tests give feedback indicating that this isn't possible currently. It might be that it is simply a better idea just having ```x64``` supported, and then sometime later add support for ```x86```.

Reply:
Yes you're right. Now we know 😄 
I agree with you about having only x64 for now.

Reply:
Let's note that `winget validate` did not catch that on my end...